docs/guides/routing.rst

```diff
@@ -92,6 +92,10 @@ The following conventions are used in core and recommended for plugins:
 		- ``collection:object:blog:friends``: list blogs owned by friends of the logged in user (or user with a given username)
 		- ``collection:object:blog:group``: list blogs in a group
 
+**collection:<entity_type>:<entity_subtype>**
```

Comment:
Of course Elgg maps `/blog` to the collection All Blogs, but it may not be a collection. A plugin could map it to a particular site blog or some other resource. So maybe `default:object:blog` would be a better route name?